### DISCUSSION TITLE
Impaginazione documenti scrutini

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#520] [Richieste di aiuto] [answered]
Buon pomeriggio, volevo sapere se fosse possibile, in qualche modo, ridimensionare i margini dei documenti prodotti per lo scrutinio finale.
Abbiamo modificato dati, intestazioni e firme e l'intestazione viene molto separata dalla parte alta della pagina...c'è un modo per farvi essere meno spazio?
Grazie.

**Comment by trinko** [comment, marked as answer]
No, attualmente l'unico modo sarebbe modificare direttamente il codice di alcuni file PHP.
